WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt

```diff
@@ -163,6 +176,15 @@ class HelpActivity : LocaleAwareActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    private fun launchSupportWidget() {
+        openChatWidget.launch(
+            ChatDetails(
+                selectedSiteFromExtras,
+                "https://appassets.androidplatform.net/assets/support_chat_widget.html"
```

Comment:
We can use `https://$DEFAULT_DOMAIN/assets/support_chat_widget.html` here, as like you did in `SupportWebViewActivity`.

Reply:
This is temporary, was just playing whether need to pass or use it direct, since it is. a static url.  Will update as necessary in the next PRs